src/Model.h

```diff
@@ -12,7 +12,7 @@
 
 #pragma pack(push, 2)
 
-typedef struct PERSON {
+struct person {
```

Comment:
I've just merged #252 which does this globally and calls it `struct Person`.  You'll need to resolve these conflicts.